convert2rhel/checks.py

```diff
@@ -57,6 +60,71 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_convert2rhel_latest():
+    """Make sure that we a running the latest version of convert2rhel"""
+
+    # get lastest avalible version
+
+    repo_content = (
+        "[convert2rhel]\n"
+        "name=Convert2RHEL Repository\n"
+        "baseurl=https://ftp.redhat.com/redhat/convert2rhel/$releasever/os/\n"
+        "gpgcheck=0\n"
+        "enabled=1\n"
+    )
+
+    repo_dir = os.path.join(utils.TMP_DIR, "convert2rhel_repo")
+    # repo_dir = os.path.join("/tmp/","convert2rhel_repo")
+    repo_path = os.path.join(repo_dir, "convert2rhel.repo")
+    store_content_to_file(filename=repo_path, content=repo_content)
+
+    # repoquery --setopt=reposdir=$(pwd)/test_repo --releasever=8 convert2rhel
+    raw_output_convert2rhel_versions, _ = run_subprocess(
+        ("repoquery --releasever={releasever} --setopt=reposdir={repodir} convert2rhel").format(
+            releasever=system_info.releasever,
+            # releasever="8",
+            repodir=repo_dir,
+        ),
+        print_output=False,
+    )
+
+    #   - if cannot access url: print warning and continue
+    PKG_NEVR = r"\b(\S+)-(?:([0-9]+):)?(\S+)-(\S+)\b"
+    convert2rhel_versions = re.findall(PKG_NEVR, raw_output_convert2rhel_versions, re.MULTILINE)
+
+    latest_version = ("0", "0.00", "0")
+    #   - get latest version from https://ftp.redhat.com/redhat/convert2rhel/.
+    for package_version in convert2rhel_versions:
+        if package_version[0] != "convert2rhel":
+            continue
+
+        ver_compare = rpm.labelCompare(package_version[1:], latest_version)
+
+        if ver_compare == 1:
+            latest_version = package_version[1:]
+
+    # get version of currently running convert2rhel
+
+    # if current version is not latest:
+    ver_compare = rpm.labelCompare(("0", convert2rhel.__version__, "0"), ("0", latest_version[1], "0"))
```

Comment:
```suggestion
    ver_compare = rpm.labelCompare(("0", convert2rhel_version, "0"), ("0", latest_version[1], "0"))
```

Following my suggestion from the import